### PR TITLE
New Entry: General MFA

### PR DESCRIPTION
**Pull request by yangc95**
### Description

Created entry for MFA. 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/mfa/mfa.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - 'paths/computer-science'
+---
+
+**Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.
+
+## Factors
+
+One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require user's to create when initially signing up for an account.
+
+Other factors, depending on the verification system, can be a physical item, pin, key or token, biometric marker, or location.
```
Comment (Dusch4593) [inline content/general/concepts/mfa/mfa.md:21]: Wondering if we can turn this into an unordered list?

```suggestion
Other factors, depending on the verification system, include the following:

- A physical item belonging to the user (e.g., a mobile phone).
- A password, personal identity number (PIN), key, or token that only the user should know.
- Some biometric marker for the user (fingerprints, voice-recognition, key-stroke patterns, etc.).
- Information about the user's [geolocation](https://www.codecademy.com/resources/docs/general/geolocation).
```

File: content/general/concepts/mfa/mfa.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Examples
+
+ATMs often require a physical card and a memorized pin to successfully take out cash.
```
Comment (Dusch4593) [inline content/general/concepts/mfa/mfa.md:25]:
```suggestion
ATMs often require a physical card and a memorized PIN to successfully withdraw cash.
```

File: content/general/concepts/mfa/mfa.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+A secure sites, like GitHub, allow users to enable MFA through a pin that's sent to a physical device like a mobile phone. This is very secure because in order to be verified, a person would need the physical device which probably has a pin and biometric marker factor.
+
+More factors in the verification process results in greater security.
```
Comment (Dusch4593) [inline content/general/concepts/mfa/mfa.md:29]:
```suggestion
More factors in the verification process results in greater [security](https://www.codecademy.com/resources/docs/general/cybersecurity).
```

File: content/general/concepts/mfa/mfa.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+Title: 'MFA'
+Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
```
Comment (Dusch4593) [inline content/general/concepts/mfa/mfa.md:6]:
```suggestion
  - 'Computer Science'
  - 'Cybersecurity'
```

Wouldn't this concept more closely related to the field of cybersecurity as opposed to web development?

File: content/general/concepts/mfa/mfa.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Tags:
+  - 'Best Practices'
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
```
Comment (Dusch4593) [inline content/general/concepts/mfa/mfa.md:12]:
```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/full-stack-engineer-career-path'
```

Both of the these pages contain lessons and modules pertaining to user authentication/authorization.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yangc95 Finished with my review and left you some comments/suggestions.

File: content/general/concepts/mfa/mfa.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+Title: 'MFA'
```
[the remark this answers is not in the document]
Comment (yangc95) [reply content/general/concepts/mfa/mfa.md:2]: Can do~

**Comment by yangc95** [comment]
@Dusch4593 thanks for your patience! Finally made some edits, please let me know if anything else needs tweaking~

File: content/general/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Title: 'Multi-factor Authentication'
+Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Best Practices'
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/code-foundations'
```
Comment (Dusch4593) [inline content/general/concepts/multi-factor-authentication/multi-factor-authentication.md:11]:
```suggestion
  - 'introduction-to-cybersecurity'
```

I would add at least one free course/path if possible. This one seemed like the most relevant.

**Review by Dusch4593 — APPROVED** [review]
@yangc95 LGTM! I have just one minor suggestion but we can proceed to the next review. 😄

**Comment by yangc95** [comment]
> @yangc95 LGTM! I have just one minor suggestion but we can proceed to the next review. 😄

Thanks @Dusch4593 for the suggestion!

File: content/general/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/computer-science'
+---
+
+**Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.
+
+## Factors
+
+One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require user's to create when initially signing up for an account.
```
Comment (SSwiniarski) [inline content/general/concepts/multi-factor-authentication/multi-factor-authentication.md:19]:
```suggestion
One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
```

File: content/general/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+Other factors, depending on the verification system, include the following:
+
+- A physical item belonging to the user (e.g., a mobile phone).
+- A password, personal identity number (PIN), key, or token that only the user should know.
```
Comment (SSwiniarski) [inline content/general/concepts/multi-factor-authentication/multi-factor-authentication.md:24]: You already mentioned "password." Why not mention authentication codes that are e-mailed or texted or phoned in to the user.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@yangc95, I did a review and only had a couple of comments.

File: content/general/concepts/multi-factor-authentication/multi-factor-authentication.md (hunk 9 above)
Comment (yangc95) [inline content/general/concepts/multi-factor-authentication/multi-factor-authentication.md:24]:
```suggestion
- A personal identity number (PIN), key, or token that only the user should know prior, or an authentication code that is e-mailed, texted, or phoned-in to the user in real-time.
```

**Comment by yangc95** [comment]
@SSwiniarski Thanks for taking a look! Please let me know if anything else needs changing

**Review by HishamT — APPROVED** [review]
LGTM!

**Review by SSwiniarski — APPROVED** [review]
Looks Good.

**Comment by yangc95** [comment]
live link: https://www.codecademy.com/resources/docs/general/multi-factor-authentication

TY cc: @Dusch4593 @HishamT @SSwiniarski

**Pull request merged**